### ISSUE TITLE
Error - Missing Layer when downloading a addon

### ISSUE THREAD

**Issue by fastie81** [home-assistant/supervisor#3209]
### Describe the issue you are experiencing

When adding ZwaveJS to MQTT it fails with the following error
`Can't install ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0: 500 Server Error for http+docker://localhost/v1.41/images/ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0/json: Internal Server Error ("layer does not exist")`

I have tried to run `ha supervisor repair` with no change in the error.

### What is the used version of the Supervisor?

supervisor-2021.09.6

### What type of installation are you running?

Home Assistant OS

### Which operating system are you running on?

Home Assistant Operating System

### What is the version of your installed operating system?

6.4

### What version of Home Assistant Core is installed?

core-2021.10.0

### Steps to reproduce the issue

1. Supervisor - Add-on Store
2. Search ZwaveJS to MQTT
3. install

### Anything in the Supervisor logs that might be useful for us?

```txt
21-10-07 20:42:06 INFO (SyncWorker_3) [supervisor.docker.interface] Downloading docker image ghcr.io/hassio-addons/zwavejs2mqtt/aarch64 with tag 0.26.0.
21-10-07 20:42:11 ERROR (SyncWorker_3) [supervisor.docker.interface] Can't install ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0: 500 Server Error for http+docker://localhost/v1.41/images/ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0/json: Internal Server Error ("layer does not exist")
21-10-07 20:57:30 INFO (MainThread) [supervisor.api.middleware.security] /network/info access from core_ssh
21-10-07 20:57:30 INFO (MainThread) [supervisor.api.middleware.security] /network/info access from core_ssh
21-10-07 20:57:30 INFO (MainThread) [supervisor.api.middleware.security] /host/info access from core_ssh
21-10-07 20:57:30 INFO (MainThread) [supervisor.api.middleware.security] /core/info access from core_ssh
21-10-07 20:57:33 INFO (MainThread) [supervisor.api.middleware.security] /supervisor/repair access from core_ssh
21-10-07 20:57:33 INFO (MainThread) [supervisor.core] Starting repair of Supervisor Environment
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Prune stale containers
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Prune stale images
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Prune stale builds
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Prune stale volumes
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Prune stale networks
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Fix stale container on hassio network
21-10-07 20:57:33 INFO (SyncWorker_1) [supervisor.docker] Fix stale container on host network
21-10-07 20:57:34 INFO (MainThread) [supervisor.addons] Found 0 add-ons to repair
21-10-07 20:57:34 INFO (MainThread) [supervisor.supervisor] Repairing Supervisor 2021.09.6
21-10-07 20:57:34 INFO (MainThread) [supervisor.core] Finished repair of Supervisor Environment
21-10-07 20:58:24 INFO (SyncWorker_2) [supervisor.docker.interface] Downloading docker image ghcr.io/hassio-addons/zwavejs2mqtt/aarch64 with tag 0.26.0.
21-10-07 20:58:26 ERROR (SyncWorker_2) [supervisor.docker.interface] Can't install ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0: 500 Server Error for http+docker://localhost/v1.41/images/ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0/json: Internal Server Error ("layer does not exist")
```

**Comment by ludeeus** [comment]
Run `ha su repair`

**Comment by fastie81** [comment]
HI 
Is this not the same as `ha supervisor repair`? how ever I have tried again and still same error.

`21-10-08 06:33:54 INFO (MainThread) [supervisor.core] Starting repair of Supervisor Environment
21-10-08 06:33:54 INFO (SyncWorker_5) [supervisor.docker] Prune stale containers
21-10-08 06:33:54 INFO (SyncWorker_5) [supervisor.docker] Prune stale images
21-10-08 06:33:54 INFO (SyncWorker_5) [supervisor.docker] Prune stale builds
21-10-08 06:33:56 INFO (SyncWorker_5) [supervisor.docker] Prune stale volumes
21-10-08 06:33:56 INFO (SyncWorker_5) [supervisor.docker] Prune stale networks
21-10-08 06:33:56 INFO (SyncWorker_5) [supervisor.docker] Fix stale container on hassio network
21-10-08 06:33:56 INFO (SyncWorker_5) [supervisor.docker] Fix stale container on host network
21-10-08 06:33:56 INFO (MainThread) [supervisor.addons] Found 0 add-ons to repair
21-10-08 06:33:56 INFO (MainThread) [supervisor.supervisor] Repairing Supervisor 2021.09.6
21-10-08 06:33:56 INFO (MainThread) [supervisor.core] Finished repair of Supervisor Environment
21-10-08 06:34:22 INFO (SyncWorker_3) [supervisor.docker.interface] Downloading docker image ghcr.io/hassio-addons/zwavejs2mqtt/aarch64 with tag 0.26.0.
21-10-08 06:34:29 ERROR (SyncWorker_3) [supervisor.docker.interface] Can't install ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0: 500 Server Error for http+docker://localhost/v1.41/images/ghcr.io/hassio-addons/zwavejs2mqtt/aarch64:0.26.0/json: Internal Server Error ("layer does not exist")`

**Comment by fastie81** [comment]
Hi 
Tried that, but still no luck. :(
Is there away I can install the previous version? my zwave network is down and really need to get it back up.
Thanks for yo help on this one btw.

Chris

**Comment by ludeeus** [comment]
Since you now have a fresh partial backup. tey this:

Uninstall the add-on (if installed) -> reboot the host  -> `ha su repair` -> reboot the host -> do a partial restore to only restore the add-on

**Comment by fastie81** [comment]
Hi 
As I lost the add-on before I got the backup I dont have a partial backup. sorry that I did not made that clear before.
So went a did what you suggested other than restoring from the partial backup, in a hope it will install it. 
Is there anything ells I can do or try? sorry about me not being clear at my state.

**Comment by fastie81** [comment]
Hi,
So I managed to get version 0.25.1 of the add-on installed and I have the ZwaveJS on my sidebar. I have tested my zwave network and I can get info from my devices and switch some on and off via the ZwaveJS Control Panel. 
HA was still not showing my devices, so I looked at the integration and its now showing me and error of 
`Retrying setup: Z-Wave JS Server version is incompatible: 1.10.4 a version is required that supports at least api schema 10`
From what I understand this is due to the version difference between the integration and the add-on and all the suggestions I can find suggest to update the add-on.
Anyway I can fix this issue? I still cant update the add-on to 0.26.0 as it still giving me the missing layer error.

Any advice would be much appreciated
